worker/orca_grader/__main__.py

```diff
@@ -145,6 +145,8 @@ def run_grading_job(grading_job: GradingJobJSON, no_container: bool,
             handle_grading_job(grading_job, image_name, container_command)
         else:
             handle_grading_job(grading_job, image_name)
+    except InvalidWorkerStateException as ive:
```

Comment:
Where do you get the initials "i v e" from Invalid Worker State Exception?

Reply:
Probably ought to LOGGER.critical this error?  Or do you log every time you raise such an error?

Reply:
It gets logged at the `process_db_jobs` function, which this bubbles up to